purchases/src/test/java/com/revenuecat/purchases/paywalls/components/properties/MaskShapeTests.kt

```diff
@@ -4,9 +4,9 @@ import com.revenuecat.purchases.common.OfferingParser
 import org.intellij.lang.annotations.Language
 import org.junit.Test
 import org.junit.runner.RunWith
-import org.junit.runners.Parameterized
+import org.robolectric.ParameterizedRobolectricTestRunner
 
-@RunWith(Parameterized::class)
+@RunWith(ParameterizedRobolectricTestRunner::class)
```

Comment:
Had to change the runner because I added the log when there is an error deserializing cornerradiuses and that was failing without robolectric.